controllers/rhmi/rhmi_controller.go

```diff
@@ -1464,3 +1466,16 @@ func getInstallation() (*rhmiv1alpha1.RHMI, error) {
 		},
 	}, nil
 }
+
+func userTenantAnnotationPredicate(installation *rhmiv1alpha1.RHMI) predicate.Predicate {
+	return predicate.NewPredicateFuncs(func(m metav1.Object, _ runtime.Object) bool {
+		//if !rhmiv1alpha1.IsRHOAMMultitenant(rhmiv1alpha1.InstallationType(installation.Spec.Type)) {
```

Comment:
can we remove comments?

Reply:
Done. Also this method removed, not needed already , as this additional Watches was removed, as suggested in next comment